ci/docker/base.test.Dockerfile

```diff
@@ -44,14 +52,11 @@ ENV LC_ALL=en_US.utf8
 ENV LANG=en_US.utf8
 RUN echo "ulimit -c 0" >> /root/.bashrc
 
-# Setup Bazel caches
-RUN echo "build --remote_cache=${REMOTE_CACHE_URL}" >> /root/.bazelrc
-
```

Comment:
`ci/env/install-bazel.sh` will always fully rewrite the `.bazelrc` file, so there is actually no point to write it here.